### PR TITLE
Ticket 7

### PR DESCRIPTION
**Pull request by emmanuel-irandi**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -99,3 +99,15 @@ describe("GET comments", ()=>{
...
+        return request(app)
+        .post("/api/articles/1/comments")
+        .send({username : "lurker", body : "unsubing rn"})
+        .expect(201)
+        .then((responce) => {
+            expect(responce.body).toHaveProperty("username")
+            expect(responce.body).toHaveProperty("body")
+        })
+    })
+});
```
Comment (nc-mark-t) [inline __tests__/app.test.js:113]: We need to handle the edge cases too. 
- what if the body is missing
- if the username doesn't exist
- if the article doesn't exist
- if the article_id is invalid

File: controllers/postCommentController.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const insertComment = require("../models/postCommentModel.js");
+
+const postComment = (req,res)=>{
+    console.log(req.params.article_id,req.body)
```
Comment (nc-mark-t) [inline controllers/postCommentController.js:4]: Try to remove console.logs before you submit Prs

**Review by nc-mark-t — COMMENTED** [review]
You happy path is great, now you need to consider the errors
